packages/udaru-hapi-plugin/package.json

```diff
@@ -35,9 +35,15 @@
   "scripts": {
     "coverage": "UDARU_SERVICE_logger_pino_level=silent lab -c -t 96 -r html -o docs/coverage.html",
     "coveralls": "UDARU_SERVICE_logger_pino_level=silent lab -c -t 96 -r lcov | COVERALLS_REPO_TOKEN='?' coveralls",
-    "test": "UDARU_SERVICE_logger_pino_level=silent lab -c -t 96"
+    "depcheck": "npx depcheck",
+    "pg:init": "UDARU_SERVICE_local=true npx udaru-init && npm run pg:migrate",
+    "pg:init-test-db": "npm run pg:init && npm run pg:load-test-data",
+    "pg:load-test-data": "UDARU_SERVICE_local=true npx udaru-loadTestData",
+    "pg:migrate": "npx udaru-migrate --version=7",
```

Comment:
should we use `max` for migration version?

Reply:
How do you mean exactly here Mihai?

Reply:
postgrator has an option to put `--version=max` and will migrate to the highest level (in our case 7) so we don't have to manually update this number on each new migration added

Reply:
ah great, yes let me change that